static/app/views/performance/transactionSummary/transactionVitals/vitalsPanel.tsx

```diff
@@ -21,6 +19,7 @@ type Props = {
   eventView: EventView;
   location: Location;
   organization: Organization;
+  results: Object;
```

Comment:
`object` here might work better since it seems that you're referring to a [type](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html) and not the `Object` constructor

Reply:
Oh nice, great catch! Thanks @udameli